namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ThriftNamerInterface.scala

```diff
@@ -73,7 +74,11 @@ object ThriftNamerInterface {
    * A utility for generating stamps unique to an instance.
    */
   private[namerd] class LocalStamper extends Stamper {
```

Comment:
TIOLI: this would be a bit more testable if we passed in the Random as a constructor argument.  Not that it really matters since we don't currently have good tests for this 🙃